src/styles/sciml/nest.jl

```diff
@@ -183,7 +183,7 @@ for f in [
     :n_curly!,
     :n_macrocall!,
     :n_ref!,
-    :n_vect!,
+    # :n_vect!, don't use YAS style vector formatting with `yas_style_nesting = true`
```

Comment:
With `yas_style_nesting = true`, this just restores the original version before #792.

With `yas_style_nesting = false`, #792 forwarded `n_vect!` to `_n_tuple!` below. Now it is forwarded to the default style `n_vect!`. Is this correct?

Reply:
This would change the nesting behaviour imposed in #792 so that it would put all arguments on one line instead of trying to best fit the arguments among the fewest lines while trying to fit within the maximum margin.

The entire point of #792 was that the former behaviour was unwanted amongst people in the SciML organization.

Reply:
for vect specifically if this is fine then I'm ok with the PR but I'm not sure if there's a concensus on it.

Reply:
I see. I guess we want to best fit, but still allow the line break with vector definitions. I'll have another look tomorrow. Or can you point me in the right direction how to do that?

Reply:
> but still allow the line break with vector definitions

can you give an example of what this means?

Reply:
Sorry, I meant that I would like to combine the vector definition style that I restored in this PR with the best fit feature you implemented.